java/src/jmri/Manager.java

```diff
@@ -865,6 +865,13 @@ public default String getEntryToolTip() {
         return null;
     }
 
+    /**
+     * @return the interval (delay) in Ms between output commands for the manager.
+     */
+    public static int getOutputInterval(@Nonnull String SystemName) {
```

Comment:
What does SystemName here mean?  Why is that an argument? (It should also be initial-lower-case) 

And why is this here, instead of just being in TurnoutManager?